auslib/blobs/base.py

```diff
@@ -55,6 +55,41 @@ def createBlob(data):
     return blob_map[schema_version](**data)
 
 
+def merge_dicts(ancestor, left, right):
```

Comment:
Reading through this function, there are lots of places where you're looking at all ancestor, left and right.

Would having these 3 in a variable help clean the code up?

`
dicts = (ancestor, left, right)
for key in set(key for d in dicts for key in d.keys()):
...
`

and

`
if any(isinstance(d.get(key), dict) for d in dicts):
...
`

Reply:
Seems like it should - I'll give it a shot.